src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInternalConnectionTds.cs

```diff
@@ -1237,6 +1236,108 @@ private bool IsTransientError(SqlException exc)
             return false;
         }
 
+        private void OpenLoginEnlist(
+            TimeoutTimer timeout,
+            SqlConnectionString connectionOptions,
+            SqlCredential credential,
+            string newPassword,
+            SecureString newSecurePassword,
+            bool redirectedUserInstance)
+        {
+            // Indicates whether we should use primary or secondary first
+            bool useFailoverPartner;
+            string failoverPartner;
```

Comment:
These two were originally defined with the following if statement in between them. I moved them to be closer together since they are related.